frontends/ol-components/src/components/LearningResourceExpanded/LearningResourceExpandedV2.tsx

```diff
@@ -194,11 +207,68 @@ const OnPlatform = styled.span({
   color: theme.custom.colors.black,
 })
 
-const ListButtonContainer = styled.div({
+const ButtonContainer = styled.div({
   display: "flex",
+  width: "100%",
   gap: "8px",
   flexGrow: 1,
-  justifyContent: "flex-end",
+  justifyContent: "center",
+})
+
+const SelectableButton = styled(Button)<{ selected?: number }>((props) => [
```

Comment:
Could you make `selected` a boolean?

Reply:
I tried this previously and it wasn't working, but seems to be working now. Previously, it was complaining about it being a boolean because React was writing the props as HTML attributes. This was probably because I was passing `selected` by itself, rather than with property expansion.